ninja-servlet/src/main/java/ninja/servlet/NinjaBootstrap.java

```diff
@@ -191,7 +206,15 @@ protected void configureServlets() {
             return injector;
 
         } catch (Exception exception) {
-            logger.error("Fatal error booting Ninja", exception);
+            
+            // save exception in case other consumers of this class would like
+            // to handle an injector exception a little differently
+            injectorException = exception;
```

Comment:
Why is that necessary? Would it not be enough to simply catch the exception one level above this class? I am just asking because all the isInjector... getInjectorException... etc makes a lot of noise in different classes.